esmvaltool/cmorizers/data/formatters/datasets/esacci_biomass.py

```diff
@@ -0,0 +1,174 @@
+"""ESMValTool CMORizer for ESACCI-BIOMASS above-ground biomass (agb) data.
+
+Tier
+    Tier 2: other freely-available dataset.
+
+Source
+    ftp://anon-ftp.ceda.ac.uk/neodc/esacci/biomass/data/agb/maps
+
+Last access
+    20250716
+
+Download and processing instructions
+    Download 10 km file:
+      v6.0/netcd/ESACCI-BIOMASS-L4-AGB-MERGED-10000m-fv6.0.nc
+    Put file in ${RAWOBS}/Tier2/ESACCI-BIOMASS
```

Comment:
Also mention the automatic downloading script here?

Reply:
Good idea. Done (https://github.com/ESMValGroup/ESMValTool/pull/4121/commits/71e451be5ef2da679a38e328fdfc8150b6007a63).